### ISSUE TITLE
Segmentation fault when running google-drive-ocamlfuse (xubuntu 14.04)

### ISSUE THREAD

**Issue by aranc** [astrada/google-drive-ocamlfuse#127]
System: xubuntu 14.04 + firefox

When running google-drive-ocamlfuse (from ppa or from cloned git), firefox is opened and the program crash.

```
aranc@aranc:~/google-drive-ocamlfuse$ ./gdfuse.native 

(process:2607): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed
Segmentation fault (core dumped)
```

When running with gdb:
```
aranc@aranc:~/google-drive-ocamlfuse$ gdb ./gdfuse.native 
Reading symbols from ./gdfuse.native...done.
(gdb) run
Starting program: /home/aranc/google-drive-ocamlfuse/gdfuse.native 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/i386-linux-gnu/libthread_db.so.1".
[New Thread 0xb741bb40 (LWP 2805)]

(process:2806): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed
[Thread 0xb741bb40 (LWP 2805) exited]

Program received signal SIGSEGV, Segmentation fault.
0xb7b88962 in ?? () from /lib/i386-linux-gnu/libgcrypt.so.11
(gdb) bt
#0  0xb7b88962 in ?? () from /lib/i386-linux-gnu/libgcrypt.so.11
#1  0xb7b89b8d in ?? () from /lib/i386-linux-gnu/libgcrypt.so.11
#2  0xb7b8a382 in ?? () from /lib/i386-linux-gnu/libgcrypt.so.11
#3  0xb7b89dd1 in ?? () from /lib/i386-linux-gnu/libgcrypt.so.11
#4  0xb7b8a224 in ?? () from /lib/i386-linux-gnu/libgcrypt.so.11
#5  0xb7b6f19e in ?? () from /lib/i386-linux-gnu/libgcrypt.so.11
#6  0xb7b657c4 in gcry_cipher_setkey () from /lib/i386-linux-gnu/libgcrypt.so.11
#7  0xb7b3460b in ?? () from /usr/lib/i386-linux-gnu/libgnutls.so.26
#8  0xb7ab9849 in ?? () from /usr/lib/i386-linux-gnu/libgnutls.so.26
#9  0xb7ac4a79 in ?? () from /usr/lib/i386-linux-gnu/libgnutls.so.26
#10 0xb7ac5217 in ?? () from /usr/lib/i386-linux-gnu/libgnutls.so.26
#11 0xb7ac57fb in ?? () from /usr/lib/i386-linux-gnu/libgnutls.so.26
#12 0xb7aae2f5 in ?? () from /usr/lib/i386-linux-gnu/libgnutls.so.26
#13 0xb7ab17af in ?? () from /usr/lib/i386-linux-gnu/libgnutls.so.26
#14 0xb7ab3164 in gnutls_handshake () from /usr/lib/i386-linux-gnu/libgnutls.so.26
#15 0xb7ed0422 in ?? () from /usr/lib/i386-linux-gnu/libcurl-gnutls.so.4
#16 0xb7ed1461 in ?? () from /usr/lib/i386-linux-gnu/libcurl-gnutls.so.4
#17 0xb7ed20c8 in ?? () from /usr/lib/i386-linux-gnu/libcurl-gnutls.so.4
#18 0xb7e9320c in ?? () from /usr/lib/i386-linux-gnu/libcurl-gnutls.so.4
#19 0xb7ea24f8 in ?? () from /usr/lib/i386-linux-gnu/libcurl-gnutls.so.4
#20 0xb7eb5dea in ?? () from /usr/lib/i386-linux-gnu/libcurl-gnutls.so.4
#21 0xb7eb6b10 in curl_multi_perform () from /usr/lib/i386-linux-gnu/libcurl-gnutls.so.4
#22 0xb7ead96b in curl_easy_perform () from /usr/lib/i386-linux-gnu/libcurl-gnutls.so.4
#23 0x0819ddb0 in helper_curl_easy_perform ()
#24 0x080ee5fb in camlCurl__perform_1368 ()
#25 0x0807dc5b in camlGapiConversation__request_1095 () at src/gapi/gapiConversation.ml:207
#26 0x0806d519 in camlGaeProxy__fun_1124 () at src/gaeProxy.ml:15
#27 0x0807de64 in camlGapiConversation__with_session_1129 () at src/gapi/gapiConversation.ml:262
---Type <return> to continue, or q <return> to quit---l
#28 0x0807df32 in camlGapiConversation__with_curl_1147 () at src/gapi/gapiConversation.ml:277
#29 0x0806d816 in camlGaeProxy__get_tokens_1082 () at src/gaeProxy.ml:69
#30 0x0806d5b6 in camlGaeProxy__loop_1090 () at src/gaeProxy.ml:89
#31 0x08055d64 in camlGdfuse__get_auth_tokens_from_server_1094 () at src/gdfuse.ml:113
#32 0x08058050 in camlGdfuse__entry () at src/gdfuse.ml:506
#33 0x0804dca5 in caml_program ()
#34 0x081c8136 in caml_start_program ()
#35 0x081b773a in caml_main ()
#36 0x081b777c in main ()
(gdb) l
1	events.c: No such file or directory.
```

**Comment by aranc** [comment]
libgcrypt11 version is 1.5.3-2ubuntu4.2

could be a duplicate of https://github.com/astrada/google-drive-ocamlfuse/issues/106

**Comment by astrada** [comment]
> libgcrypt11 version is 1.5.3-2ubuntu4.2
> could be a duplicate of #106

I think so. It should be fixed in libgcrypt11 1.5.4.
